packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,63 @@ section: get-started
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>December 2024</Timestamp>
+
+## PatternFly 6.1
+
+### Promoted package versions
+
+### Component updates 
+
```

Comment:
Here are the core PRs that will go in the patch release. I can review these with you and help write descriptions, just lemme know!

https://github.com/patternfly/patternfly/pull/7246
https://github.com/patternfly/patternfly/pull/7217
https://github.com/patternfly/patternfly/pull/7198
https://github.com/patternfly/patternfly/pull/7184
https://github.com/patternfly/patternfly/pull/7224
https://github.com/patternfly/patternfly/pull/7175
https://github.com/patternfly/patternfly/pull/7189
https://github.com/patternfly/patternfly/pull/7201
https://github.com/patternfly/patternfly/pull/7213
https://github.com/patternfly/patternfly/pull/7176
https://github.com/patternfly/patternfly/pull/7218
https://github.com/patternfly/patternfly/pull/7212
https://github.com/patternfly/patternfly/pull/7134
https://github.com/patternfly/patternfly/pull/7087
https://github.com/patternfly/patternfly/pull/7171
https://github.com/patternfly/patternfly/pull/7152